### PR TITLE
Added example to vignette #13 

### PR DESCRIPTION
**Pull request by fontikar**
In reference to #13 
Added an example of using smatr - used example for sma()
Removed old/redundant articles/RMD

### CODE REVIEW

File: .github/workflows/pkgdown.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ on:
     branches:
       - main
       - master
+      - vign
```
Comment (dfalster) [inline .github/workflows/pkgdown.yml]: Generally want to remove this before sending PR

File: vignettes/smatr.Rmd [hunk 2]
```diff
@@ -15,5 +15,176 @@ knitr::opts_chunk$set(
 ```
 
 ```{r setup}
+#Load library
```
Comment (dfalster) [inline vignettes/smatr.Rmd]: For a vignette, we want to separate the code below into several chunks, adding some of the text in comments as text in the document, and removing the results. The results will be generated when the vignette is built.

**Review by dfalster — CHANGES_REQUESTED** [review]
Hi Fonti. Thanks for the PR! Can you make the requested changes to vignette?

File: .github/workflows/pkgdown.yml (hunk 1 above)
Comment (fontikar) [reply .github/workflows/pkgdown.yml]: Just removed with last commit 2444895
 Good tip for future PR, such a handy way to test!

File: vignettes/smatr.Rmd (hunk 2 above)
Comment (fontikar) [reply vignettes/smatr.Rmd]: Added more chunks and text, mostly copy and pastes from help files and paper 3492bcbe13b827450a2d1400e54ff2ea236f4b27  I think any detailed theory or specifics I will have to leave to creators!

**Pull request merged**